.pre-commit-config.yaml

```diff
@@ -3,47 +3,58 @@ ci:
   skip: [hadolint-docker]
 
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.0.1
+  # Autoformat: Python code
+  - repo: https://github.com/ambv/black
+    rev: 21.6b0
     hooks:
-      - id: check-yaml
-        files: .*\.(yaml|yml)$
+      - id: black
+        args: [--target-version=py39]
```

Comment:
Once again, let's put black and flake8 close to each other in this file.

Reply:
Oh, first i thought you meant that, then I thought you meant the version.

Reply:
I meant both and there were two comments :)